conf/ini-files/DEFAULTS.ini

```diff
@@ -381,7 +381,7 @@ PAHDB                       = http://www.pahdb.mcgill.ca/PahdbSearch.php?SearchV
 PHARMGKB                    = https://www.pharmgkb.org/rsid/###ID###
 PHENCODE                    = http://phencode.bx.psu.edu/cgi-bin/phencode/phencode?build=hg19&id=###ID### 
 QUICK_GO_IMP                = http://www.ebi.ac.uk/QuickGO/GAnnotation?goid=###ID###&evidence=IMP&protein=###PR_ID###
-RGD_SEARCH                  = http://rgd.mcw.edu/rgdweb/report/reference/main.html?id=###ID###
+RGD_SEARCH                  = https://rgd.mcw.edu/rgdweb/elasticResults.html?category=###TYPE###&term=###ID###
```

Comment:
Hi Laurent,
can you try setting this to
://rgd.mcw.edu/rgdweb/elasticResults.html?category=###TYPE###&term=###ID### and see if it works